ssn/rdf/sosa.ttl

```diff
@@ -168,6 +168,38 @@ A Sample is intended to sample some FatureOfInterest in an application domain, s
   skos:example "A statistical sample is often designed to be characteristic of an entire population, so that observations can be made regarding the sample that provide a good estimate of the properties of the population."@en ;
   skos:note "A transient sample, such as a ships-track or flight-line, might be identified and described, but is unlikely to be revisited exactly."@en ;
 .
+sosa:Sampler
+  rdf:type rdfs:Class ;
+  rdf:type owl:Class ;
+  rdfs:label "Sampling device"@en ;
+  rdfs:subClassOf owl:Thing ;
+  skos:definition "A device that is used by, or implements, a (Sampling) Procedure to create or transform one or more samples."@en ;
+  skos:example "Ball mill"@en ;
+  skos:example "Diamond drill"@en ;
+  skos:example "Hammer"@en ;
+  skos:example "Hypodermic syringe and needle"@en ;
+  skos:example "Image sensor"@en ;
+  skos:example "Seive stack"@en ;
+  skos:example "Soil auger"@en ;
```

Comment:
I disagree. More examples are better. We should increase the number elsewhere rather than cull them anywhere.